plotly/tools.py

```diff
@@ -351,12 +351,17 @@ def embed(file_owner_or_url, file_id=None, width="100%", height=525):
     Keyword arguments:
     file_id (default=None) -- an int or string that can be converted to int
                               if you're using a url, don't fill this in!
+    share_key (default=None) -- a string that enables private plots to be
+                                shared. Note, you must have admin privileges
+                                on the file for this to work
     width (default="100%") -- an int or string corresp. to width of the figure
-    height (default="525") -- same as width but corresp. to the height of the figure
+    height (default="525") -- same as width but corresp. to the height of the
+                              figure
 
     """
     try:
-        s = get_embed(file_owner_or_url, file_id, width, height)
+        s = get_embed(file_owner_or_url, file_id, share_key, width, height)
```

Comment:
so just to make sure I understood correctly, we should call it like `get_embed(file_owner_or_url, file_id, share_key=share_key, width=width, height=height)` to avoid any keyword arguments?